doc/index.rst

```diff
@@ -13,6 +13,7 @@ Project 'Universum'
     code_report.rst
     github_handler.rst
     universum_docs.rst
+    internal.rst
```

Comment:
I'd actually moved this to before examples. This is not the most interesting docs for user: interaction with CodeReview systems seems more important to me.